frontend/components/outcomes/OutcomesHome.js

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Is this file a duplicate of `OutcomesScreen.js`? I couldn't find any usages of this component.